packages/fhir-quest-form/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@opensrp/fhir-quest-form",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/types",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/opensrp/web",
+  "config": {
+    "folderName": "fhir-quest-form"
```

Comment:
I think the reason we were appending 'fhir' to package names was because we had the old packages without fhir. makes sense to drop fhir from completely new packages

Reply:
I kind of disagree, I think in this case fhir adds some context as to what exactly this package offers, its not just any other form but a form that works for certain mentioned fhir-resources

Reply:
The goal was to move the whole repo to fhir so in the end the fhir context will apply to the whole repo